tsconfig.json

```diff
@@ -11,14 +11,15 @@
     "alwaysStrict": true,
     "noImplicitReturns": true,
     "strictFunctionTypes": true,
-    "allowJs": true
+    "allowJs": true,
+    "sourceMap": true
```

Comment:
missing sourceMap

Reply:
There are many ways to generate and use source maps and I'm not yet sure how we're going to go about this. Will deal with this later.